pkg/schedule/config/config.go

```diff
@@ -30,7 +30,7 @@ const (
 	DefaultMaxReplicas            = 3
 	defaultMaxSnapshotCount       = 64
 	defaultMaxPendingPeerCount    = 64
-	defaultMaxMergeRegionSize     = 20
+	defaultMaxMergeRegionSize     = 96
```

Comment:
We need to add more comments about which version changed and the related issue.

Reply:
Will it cause lots of merge operations?

Reply:
+1, 96 is too aggressive, it might cause a lot of merging after upgrading. We increase the default region size from 96MB to 256MB in TiKV side, it is 2.67x larger than before. But here 20 -> 96 is nearly 5x larger than before, probably 40MB is better.

Reply:
How about using `54MB`(=> `20MB * 256 / 96`) as the recommended value ?

Reply:
> Will it cause lots of merge operations when upgrading?

Nope, the old value of `max-merge-region-size` is already persisted in the cluster and it will be reloaded after upgrading. Only the newly created cluster will use the new default value.

Reply:
Oh, right. But for those existing clusters, do we still use the old max-merge-region-size and let the user change the config? If so, then LGTM.

Reply:
IMO, it should be.

Since each user's cluster may have a unique self-defined `max-merge-region-size`, manual adjustments should be checked and executed if the user requires it.